recipes/opensearch/build.sh

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -o xtrace -o nounset -o pipefail -o errexit
+
+export JAVA_OPTS="-XX:ReservedCodeCacheSize=64m"
+# https://opensearch.org/docs/1.0/opensearch/install/important-settings/
+export OPENSEARCH_JAVA_OPTS="-Xms512m -Xmx512m"
+sed -i "s/id 'lifecycle-base'/id 'lifecycle-base'\nid 'com.github.jk1.dependency-license-report' version 'latest.release'/" build.gradle
+
+# Build OpenSearch distribution (https://github.com/opensearch-project/OpenSearch/blob/main/DEVELOPER_GUIDE.md#gradle-build)
+./gradlew -Dbuild.noJdk=true localDistro
+
+# Download dependency licenses
+./gradlew publishToMavenLocal --warning-mode all
+
+mkdir -p "$PREFIX/bin"
+mkdir -p "$PREFIX/libexec/opensearch"
+mkdir -p "${PREFIX}/libexec/opensearch/logs"
```

Comment:
```suggestion
mkdir -p "$PREFIX/libexec/opensearch/logs"
touch "$PREFIX/libexec/opensearch/logs/gc.log"
```
It looks like `conda-build` will ignore empty directories, so adding an empty log to the directory is needed for the folder to still exist.